### DISCUSSION TITLE
Cannot create issue - Comment is too long (maximum is 65536 characters)

### DISCUSSION THREAD

**Discussion by kanehu** [community/community#41331] [Projects and Issues] [answered]
### Select Topic Area

Product Feedback

### Body

I receive this message, while the comment is filled????
![Github](https://user-images.githubusercontent.com/70489959/206851382-1989cdfd-205d-4203-9511-00e058c848f9.png)

**Comment by jfullstackdev** [comment, marked as answer]
here are the limits, check this out

https://github.com/dead-claudia/github-limits

**Reply by kanehu** [reply]
Thank you, I checked it. It is not written explicitly there, but I could figure the solution out. I pasted a logcat, which was longer than 65536 characters, and that was the problem. So thanks again.

**Reply by kishaningithub** [reply]
@jdevstatic even though the error message says maximum is 65536 characters (~65 KB) i am actually able to pass a ~200 KB text file and only beyond this the API fails with this max chars failure. Should the documentation be updated or what do you think i am missing?

Reference
- https://github.com/mshick/add-pr-comment/issues/93#issuecomment-1531415467

**Reply by haguenau** [reply]
From inspecting request in a Web browser's developer tools, it appears that the 65536-character limit is based on the API call's gzipped size. A 231k-character comment I was able to post successfully resulted in a 55K API call.

That still is a pretty misleading error message; it should either clearly say what the limit applies to, or the error should trigger based on uncompressed body length.

**Reply by Julian-Dumitrascu** [reply]
@haguenau We're moving to GitLab. A difference is that they limit such a message to 1 million characters.

**Comment by tr1p0p** [comment]
This is ridiculous. How can i POST LOGS on github if i got ERROR OF TOO MANY CHARACTERS ????

**Comment by amitdo** [comment]
You can attach the log files as zip archive to the issue.
